ecs/cloudwatch.tf

```
@@ -12,7 +12,7 @@ resource "aws_cloudwatch_log_group" "this" {
 module "sentinel_forwarder" {
   count             = var.sentinel_forwarder ? 1 : 0
   source            = "github.com/cds-snc/terraform-modules//sentinel_forwarder?ref=main"
-  function_name     = "${var.cluster_name}-sentinel-forwarder"
+  function_name     = "${var.cluster_name}"
```

Just to make sure we don't accidentally overflow with a long cluster name, could you also add a substring like so:
```suggestion
  function_name     = substr(var.cluster_name, 0, 64)
```

---

Thanks Pat for the suggestion (as always). This has been corrected.